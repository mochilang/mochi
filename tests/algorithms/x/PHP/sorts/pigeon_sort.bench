{
  "duration_us": 311,
  "memory_bytes": 42408,
  "name": "main"
}