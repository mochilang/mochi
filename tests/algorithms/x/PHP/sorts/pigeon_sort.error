run: exit status 255

Fatal error: Uncaught Error: pigeon_sort(): Argument #1 ($array) could not be passed by reference in /workspace/mochi/tests/algorithms/x/PHP/sorts/pigeon_sort.php:134
Stack trace:
#0 {main}
  thrown in /workspace/mochi/tests/algorithms/x/PHP/sorts/pigeon_sort.php on line 134
