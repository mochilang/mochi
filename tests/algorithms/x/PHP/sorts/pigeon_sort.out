Fatal error: Uncaught Error: pigeon_sort(): Argument #1 ($array) could not be passed by reference in /workspace/mochi/tests/algorithms/x/PHP/sorts/pigeon_sort.php:117
Stack trace:
#0 {main}
  thrown in /workspace/mochi/tests/algorithms/x/PHP/sorts/pigeon_sort.php on line 117